pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -420,6 +423,9 @@ protected boolean consumeLoop()
         _endOfPartitionGroup = messageBatch.isEndOfPartitionGroup();
         _consecutiveErrorCount = 0;
       } catch (PermanentConsumerException e) {
```

Comment:
(minor) Suggest emitting in one place for readability
```suggestion
      } catch (Exception e) {
        _ServerMetrics.addMeteredGlobalValue...
        if (e instanceof PermanentConsumerException) {
          ...
        } else {
          ...
        }
      }
```

Reply:
will follow up in separate PR if we have more metrics